ember-cli-build.js

```diff
@@ -4,7 +4,11 @@
 const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
 module.exports = function(defaults) {
-  let options = {};
+  let options = {
+    'ember-cli-babel': {
+      includePolyfill: true
```

Comment:
Just in case you did not see, this is only the `ember-cli-build.js`, not the `index.js`.

_Something_ was using the native `Promise`, which is not available in IE11. I haven't dug down what specifically. I can do though, if you like.

Reply:
Right, I do want to fix the global `Promise` thing (see https://github.com/babel/ember-cli-babel/issues/250 for some recent discussion on this), but for now I'd prefer to `app.import` a test only vendor file that does `self.Promise = self.Promise || Ember.RSVP.Promise;` (with a comment linking to that issue so we can track it down and fix it by default).